index.js

```diff
@@ -29,20 +33,104 @@ cp.parseBlock = function () {
 	return this.finishNode(node, "BlockStatement");
 };
 
+function strToModuleIdentifier(str) {
```

Comment:
Note that `let` works in Node 4 only in strict mode, but modules are always strict, so I think we can assume `let` works.